applications/luci-app-sshtunnel/htdocs/luci-static/resources/view/sshtunnel.js

```diff
@@ -0,0 +1,403 @@
+			_("Delay after a connection failure before trying to reconnect..")
+		);
+		o.placeholder = 10;
+		o.default = 10;
+		o.datatype = "uinteger";
+		o.optional = true;
+		o.modalonly = true;
+
+		o = s.taboption("advanced", form.Value, "ServerAliveCountMax", _("Server Alive Count Max"),
+			_("ServerAliveCountMax The number of server alive messages which may be sent before ssh disconnect from the server.")
+		);
+		o.placeholder = 3;
+		o.datatype = "uinteger";
+		o.optional = true;
+		o.modalonly = true;
+
+		o = s.taboption("advanced", form.Value, "ServerAliveInterval", _("Server Alive Count Max"),
+			_("ServerAliveInterval Timeout interval in seconds for keep alive.")
+		);
+		o.optional = true;
+		o.placeholder = 0;
+		o.datatype = "uinteger";
+		o.modalonly = true;
+
+		o = s.taboption("advanced", form.ListValue, "StrictHostKeyChecking", _("Strict host key checking"),
+			_("StrictHostKeyChecking Refuse to connect to hosts if it's key has changed."));
+		o.value("accept-new", _("Accept new and check if not changed"));
+		o.value("yes", _("Yes"));
+		o.value("no", _("No"));
+		o.default = "accept-new";
+		o.modalonly = true;
+
+		o = s.taboption("advanced", form.ListValue, "LogLevel", _("Log level"), "");
+		o.value("QUIET", "QUIET");
+		o.value("FATAL", "FATAL");
+		o.value("ERROR", "ERROR");
+		o.value("INFO", "INFO");
+		o.value("VERBOSE", "VERBOSE");
+		o.value("DEBUG", "DEBUG");
+		o.default = "INFO";
+		o.modalonly = true;
+
+
+		s = m.section(form.GridSection, "tunnelR", _("Remote Tunnels"),
+			_("When the connection initiated at RemoteAddress:RemotePort and then forwarded to LocalAddress:LocalPort")
+		);
+		s.anonymous = true;
+		s.addremove = true;
+		s.nodescriptions = true;
+
+		o = s.option(form.Flag, "enabled", _("Enabled"));
+		o.default = "1";
+
+		o = _addServerOption(s);
+
+		o = s.option(form.Value, "remoteaddress", _("Remote address"),
+			_("* means to accept a connection from any interface on the OpenWrt side")
+		);
+		o.datatype = 'or(host, "*")';
+		o.optional = true;
+		o.rmempty = false;
+
+		o = s.option(form.Value, "remoteport", _("Remote port"), "");
+		o.default = "22";
+		o.datatype = "port";
+		o.rmempty = false;
+
+		o = s.option(form.Value, "localaddress", _("Local address"), "");
+		o.placeholder = "192.168.1.1";
+		o.datatype = "host";
+		o.rmempty = false;
+
+		o = s.option(form.Value, "localport", _("Local port"), "");
+		o.placeholder = "80";
+		o.datatype = "port";
+		o.rmempty = false;
+
+
+		s = m.section(form.GridSection, "tunnelL", _("Local Tunnels"),
+			_("When the connection initiated at LocalAddress:LocalPort and then forwarded to RemoteAddress:RemotePort")
+		);
+		s.anonymous = true;
+		s.addremove = true;
+		s.nodescriptions = true;
+
+		o = s.option(form.Flag, "enabled", _("Enabled"));
+		o.default = "1";
+
+		o = _addServerOption(s);
+
+		o = s.option(form.Value, "localaddress", _("Local address"),
+			_("* means to accept a connection from any interface on the OpenWrt side")
```

Comment:
`_("<em>*</em> means any local address.")`

Reply:
I used "Bind IP address or hostname. <em>*</em> means to listen all interfaces <b>including public</b>." The addition about public interfaces is important for security.